guides/v2.3/graphql/reference/customer.md

```diff
@@ -283,7 +283,7 @@ Attribute |  Data Type | Description
 `city` | String | The city or town
 `company` | String | The customer's company
 `country_id` | String | The customer's country
-`custom_attributes` | CustomerAddressAttributesInput | Address custom attributes
+`custom_attributes` | CustomerAddressAttributeInput(#customerAddressAttributeInput) | Address custom attributes
```

Comment:
Add brackets around `CustomerAddressAttributeInput` so that the link will work. See line 277 as a reference.